p2p/transport/websocket/reuseport.go

```diff
@@ -0,0 +1,9 @@
+package websocket
+
+import (
+	"github.com/libp2p/go-reuseport"
+)
+
+func reuseportIsAvailable() bool {
```

Comment:
Right now this is just overhead - I added it for parity with the TCP implementation that includes checking of a `LIBP2P_TCP_REUSEPORT` environment variable to change the setting.